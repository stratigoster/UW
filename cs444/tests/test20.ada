PACKAGE foo IS
  TYPE data;
  TYPE blood_type IS (A,B,AB,O);
  TYPE data IS RECORD
    first_name : string;
    last_name : string;
    age : integer;
    bt : bloodtype;
--       ^^^^^^^^^
-- undefined type bloodtype
  END RECORD;
  mydata : data;
END foo;
