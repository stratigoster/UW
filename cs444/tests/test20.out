***ERROR***:8:20:	Uninitialized type ` bloodtype '

------
Compilation error
------
